### ISSUE TITLE
Uncaught TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

### ISSUE THREAD

**Issue by MatheusBortolon** [facebook/nuclide#177]
When opening Atom's config folder, the message was displayed.

[Enter steps to reproduce below:]

1. Go to Settings (ctrl-comma or File>Settings)
2. In the left pane, click the Open Config Folder button.

**Atom Version**: 1.0.8
**System**: Microsoft Windows 8.1 Enterprise
**Thrown From**: [nuclide-diagnostics-store](https://github.com/facebooknuclideapm/nuclide-diagnostics-store) package, v0.0.30

### Stack Trace

Uncaught TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

```
At C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\TextEventDispatcher.js:103

TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
  at TextCallbackContainer.addCallback (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\TextEventDispatcher.js:103:34)
  at TextEventDispatcher._onEvents (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\TextEventDispatcher.js:181:31)
  at TextEventDispatcher.onFileSave (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\TextEventDispatcher.js:203:19)
  at LinterAdapter._subscribeToEvent (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\LinterAdapter.js:148:37)
  at new LinterAdapter (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\LinterAdapter.js:118:10)
  at Object.consumeLinterProvider (C:\Users\matheus.bortolon\.atom\packages\nuclide-diagnostics-store\lib\main.js:105:19)
  at Provider.module.exports.Provider.provide (C:\Users\matheus.bortolon\AppData\Local\atom\app-1.0.8\resources\app.asar\node_modules\service-hub\lib\provider.js:30:52)
  at C:\Users\matheus.bortolon\AppData\Local\atom\app-1.0.8\resources\app.asar\node_modules\service-hub\lib\service-hub.js:55:26
  at process._tickCallback (node.js:357:13)

```

### Commands

```
     -0:16.3.0 window:update-available (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "atom-ng-attr-hint",
      "tree-view"
    ]
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.0
atom-yeoman, v0.3.13
emmet, v2.3.12
hyperclick, v0.0.30
json-schema, v0.1.13
linter, v1.4.3
linter-bootlint, v0.0.6
linter-coffeelint, v1.0.2
linter-jshint, v1.1.5
linter-jsonlint, v0.1.4
linter-php, v1.0.3
linter-tidy, v1.0.1
linter-xo, v0.4.0
nuclide-arcanist, v0.0.30
nuclide-code-format, v0.0.30
nuclide-diagnostics-store, v0.0.30
nuclide-diagnostics-ui, v0.0.30
nuclide-diff-view, v0.0.30
nuclide-file-tree, v0.0.30
nuclide-file-watcher, v0.0.30
nuclide-find-references, v0.0.30
nuclide-flow, v0.0.30
nuclide-hack, v0.0.30
nuclide-hg-repository, v0.0.30
nuclide-installer, v0.0.30
nuclide-language-hack, v0.0.30
nuclide-ocaml, v0.0.30
nuclide-quick-open, v0.0.30
nuclide-remote-projects, v0.0.30
nuclide-type-hint, v0.0.30

# Dev
No dev packages
```

**Comment by scv119** [comment]
Thanks for reporting the bug.

**Comment by nmote** [comment]
This should be fixed as of a922cd3142254d2bc52acf17206f2cc819c71ff3.

If you are using a development version of Nuclide, please pull. If not, this should be fixed in the next release. Please re-open the issue if this problem persists.

**Comment by MatheusBortolon** [comment]
Thanks for the quick fix, and congratulations on your great tool!
